### ISSUE TITLE
A question about Trigger Netlify Hook

### ISSUE THREAD

**Issue by ironnysh** [edjw/github-issues-headless-cms#26]
Hi @edjw ,
First, I'd like to thank you for this awesome project. Brilliant use case for 11ty!
I forked the repository, but I can't seem to get Netlify/Vercel to do a rebuild when I submit a new issue. I guess I'm missing a step in the way that has to do with the [rebuild.yml](https://github.com/ironnysh/github-issues-headless-cms/blob/master/.github/workflows/rebuild.yml) file. Would appreciate your help in pointing me to the right place to configure this option in Netlify or Vercel.

Thanks,
Ronny

**Comment by ironnysh** [comment]
Hi @edjw ,
Sorry to bother you, I figured it out with Netlify. Thanks!

**Comment by edjw** [comment]
Hi @ironnysh, great you got it working. Was there something I can explain or annotate that would have made it easier to work out?

Also interested in what you're planning on using it for?

**Comment by ironnysh** [comment]
Hi @edjw ,
It's working on Netlify, currently trying to recreate my steps to get it to work on Vercel as well (haven't decided which one I prefer :-) ).
I've been following your progress since seeing your tweet about it, and it really fascinated me to try it out. For some reason, it didn't work at all when I cloned the repo, changed the details, and pushed it to my account - had to fork to get it to properly display the content of the issue and not generate errors on deploy. I read [this post](https://github.com/edjw/github-issues-headless-cms/issues/7), and as a noob, I guess the main "mystery" were the build hooks for continuous deployment when issues are opened/updated.
I really like 11ty, but must admit I'm struggling with finding a reasonable workflow for non-tech people to use it. My partner/friends/relatives who need a little website or a blog won't use VS code for writing content, so I thought that Github's issues UI might be friendlier and more intuitive. Also, as a Hebrew-speaker, I (and all those people mentioned above) can't really use the nice Markdown apps - and most nice CRM's, for that matter - available for Left-to-Right writers :-)

**Comment by edjw** [comment]
That was why I made it so that's good!

For the rebuild.yml to work…

1. Go to https://app.netlify.com/sites/github-issues-headless-cms/settings/deploys#build-hooks

Add a build hook. Give it a name. Copy the URL

2. Go to https://github.com/ironnysh/github-issues-headless-cms/settings/secrets

Click New Secret. Give it the name NETLIFY_REBUILD. Paste in the URL from Netlify. Click Add secret.

I'm afraid I haven't used Vercel. But you probably need to make a Deploy Hook like it says here and use that in step 2 above:
https://vercel.com/docs/v2/more/deploy-hooks

**Comment by ironnysh** [comment]
Once I finish setting it up I’ll ask a WordPress diehard to give a try. I’ll report back :-)

Oh, that’s a lot less complicated than what I did :-) So I don’t have to create additional webhooks in GitHub and set Environment variables on Netlify? 
Does that mean I could theoretically add Vercel’s secret in the same rebuild.yml?

**Comment by edjw** [comment]
Yes but don’t do that. You don’t want the secret to be public. If you put it straight in the rebuild.yml file, anyone could trigger rebuilds of your site at any time. That could end up with high charges


Make the build webhook in netlify or vercel then put that in GitHub secrets for the repository. That way no one can see it
GitHub <notifications@github.com> wrote:
“Once I finish setting it up I’ll ask a WordPress diehard to give a try. I’ll report back :-)

Oh, that’s a lot less complicated than what I did :-) So I don’t have to create additional webhooks in GitHub and set Environment variables on Netlify?
 Does that mean I could theoretically add Vercel’s secret in the same rebuild.yml?”

**Comment by ironnysh** [comment]
Good point, hadn’t thought about it, thanks!
And you know if it’s possible to add Vercel’s reference in the same rebuild.yml? Cause I [duplicated](https://github.com/ironnysh/github-issues-headless-cms/blob/master/.github/workflows/vercel_rebuild.yml) your original yml file and changed it to Vercel’s details, but it totally ignores the changes to the posts (actual GitHub issues)

**Comment by edjw** [comment]
In that case, add you Vercel build we hook as a GitHub secret and give it the name ISSUES_VERCEL as that's what you used in the new rebuild.yml

**Comment by ironnysh** [comment]
Added it as a new line in the yml, but it doesn’t work. [Netlify](https://angry-williams-607850.netlify.app/) is updated but [Vercel](https://github-issues-headless-cms.vercel.app/) is frozen in time :-) 
I don’t want to take too much of your time. I’ll contact Vercel’s support team and see if they can help.
